pkg/collector/corechecks/system/cpu_windows.go

```diff
@@ -0,0 +1,160 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+// +build windows
+
+package system
+
+import (
+	"fmt"
+	"strconv"
+	"unsafe"
+
+	"github.com/DataDog/datadog-agent/pkg/autodiscovery/integration"
+	"github.com/DataDog/datadog-agent/pkg/collector/check"
+	core "github.com/DataDog/datadog-agent/pkg/collector/corechecks"
+	"github.com/DataDog/gohai/cpu"
+	log "github.com/cihub/seelog"
+	"golang.org/x/sys/windows"
+
+	"github.com/DataDog/datadog-agent/pkg/aggregator"
+)
+
+var (
+	modkernel32 = windows.NewLazyDLL("kernel32.dll")
+
+	procGetSystemTimes = modkernel32.NewProc("GetSystemTimes")
+)
+
+const cpuCheckName = "cpu"
+
+// For testing purpose
+var times = Times
+
+// TimesStat contains the amounts of time the CPU has spent performing different
+// kinds of work. Time units are in USER_HZ or Jiffies (typically hundredths of
+// a second). It is based on linux /proc/stat file.
+type TimesStat struct {
+	CPU    string
+	User   float64
+	System float64
+	Idle   float64
+}
+
+// CPUCheck doesn't need additional fields
+type CPUCheck struct {
+	core.CheckBase
+	nbCPU       float64
+	lastNbCycle float64
+	lastTimes   TimesStat
+}
+
+// Total returns the total number of seconds in a CPUTimesStat
+func (c TimesStat) Total() float64 {
```

Comment:
does the `Total` method need to be exported ?

Reply:
same